services/pub/pub-publisher.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  publisherId: Joi.string().allow(null).required(),
+}).required()
+
+class PubPublisher extends BaseJsonService {
+  static category = 'other'
+
+  static route = {
+    base: 'pub/publisher',
+    pattern: ':packageName',
+  }
+
+  static examples = [
+    {
+      title: 'Pub Publisher',
+      namedParams: { packageName: 'dart.dev' },
+      staticPreview: {
+        label: 'publisher',
+        message: 'dart.dev',
```

Comment:
Is this a valid target? :thinking: 
https://shields-staging-pr-7715.herokuapp.com/pub/publisher/dart.dev
![](https://shields-staging-pr-7715.herokuapp.com/pub/publisher/dart.dev)

![image](https://user-images.githubusercontent.com/13042488/158502190-16f3d24f-923d-4e23-b1d1-2c88d297ec7d.png)

I see it's being used for the tests and passing but something still looks off

Reply:
Its the `packageName` that is off. dart.dev is not a package: https://pub.dev/api/packages/dart.dev
How about we use `path` as the example package - that's the example you're using in the tests